spicetify.go

```diff
@@ -55,13 +54,22 @@ func init() {
 					flags = append(flags, "-"+string(char))
 				}
 			} else {
-				flags = append(flags, v)
+				if strings.Contains(v, "=") {
+					split := strings.Split(v, "=")
+					flagOption = split[1]
+					flags = append(flags, split[0])
+				} else {
+					flags = append(flags, v)
+				}
```

Comment:
This could/should later be improved as it only works when we having one single flag

A solution could be to have an array of something like maps, where the key would be the arg name (such as `--update` for eg), and the value would be the arg value (`check`)
I did not include that in this PR as it wouldn't be used for now